js/lessonfilter.js

```diff
@@ -47,74 +51,287 @@ function applySortFromURI(uri, featureList) {
 
   // Performm new sort
```

Comment:
typo

Reply:
Also there originally but happy to clean it up